### PR TITLE
Make 'MetaIngredient' get nutrients before save

### PR DESCRIPTION
**Pull request by MaiMee1**
Mini pull request for making 'MetaIngredient' get nutrients before saving. Contains some code from `command/loadrecipe`.

### CODE REVIEW

File: koocook_core/models/nutrition.py [hunk 1]
```diff
@@ -9,12 +8,6 @@
 __all__ = ['MetaIngredient', 'RecipeIngredient']
 
 
-@enum.unique
-class NutrientType(enum.Enum):
-    CARBOHYDRATE = 'carbohydrates'
-    ...
```
Comment (th-bunratta) [inline koocook_core/models/nutrition.py]: Why does this line have an ellipsis? Does it mean something?

File: koocook_core/models/nutrition.py [hunk 2]
```diff
@@ -28,6 +21,19 @@ class MetaIngredient(models.Model):
...
+    def save(self, *args, **kwargs):
+        if self.nutrient:
+            pass
+        else:
+            try:
+                from koocook_core.support.scripts import get_nutrients
+            except ModuleNotFoundError:
+                from koocook_core.management.commands._add_path import add_datatrans
+                add_datatrans()
+                from koocook_core.support.scripts import get_nutrients
```
Comment (th-bunratta) [inline koocook_core/models/nutrition.py]: Why do you import inside the function? Does it help with circular dependencies or something?

Comment (MaiMee1) [reply koocook_core/models/nutrition.py]: Prevents `datatrans` causing problems when not used

File: koocook_core/models/nutrition.py (hunk 1 above)
Comment (MaiMee1) [reply koocook_core/models/nutrition.py]: It's a deletion? It's in red. Not green.

Comment (th-bunratta) [reply koocook_core/models/nutrition.py]: OK

**Comment by plumest** [comment]
I'm OK with this it seems not effect to nutrition fact but I will check nutrition fact again after merge.

**Pull request closed without merging**